test_xblockwise_reducer.cpp:3:10: fatal error: 'xtensor/xblockwise_reducer.hpp' file not found

After building xtensor 0.24.0 for Debian, CI tests fail (testing the headers as installed in /usr/include, not the source dir):
```
[ 13%] Building CXX object CMakeFiles/test_xtensor_lib.dir/test_xcontainer_semantic.cpp.o
/usr/bin/clang++ -DXSIMD_ENABLE_XTL_COMPLEX -DXTENSOR_USE_XSIMD  -DXSIMD_ENABLE_XTL_COMPLEX=1 -march=native -std=c++14 -Wunused-parameter -Wextra -Wreorder -Wconversion -Wsign-conversion -Wold-style-cast -Wunused-variable -O3 -DNDEBUG -MD -MT CMakeFiles/test_xtensor_lib.dir/test_xbuilder.cpp.o -MF CMakeFiles/test_xtensor_lib.dir/test_xbuilder.cpp.o.d -o CMakeFiles/test_xtensor_lib.dir/test_xbuilder.cpp.o -c /tmp/autopkgtest.Gkv8A7/autopkgtest_tmp/test_xbuilder.cpp
/usr/bin/clang++ -DXSIMD_ENABLE_XTL_COMPLEX -DXTENSOR_USE_XSIMD  -DXSIMD_ENABLE_XTL_COMPLEX=1 -march=native -std=c++14 -Wunused-parameter -Wextra -Wreorder -Wconversion -Wsign-conversion -Wold-style-cast -Wunused-variable -O3 -DNDEBUG -MD -MT CMakeFiles/test_xtensor_lib.dir/test_xcontainer_semantic.cpp.o -MF CMakeFiles/test_xtensor_lib.dir/test_xcontainer_semantic.cpp.o.d -o CMakeFiles/test_xtensor_lib.dir/test_xcontainer_semantic.cpp.o -c /tmp/autopkgtest.Gkv8A7/autopkgtest_tmp/test_xcontainer_semantic.cpp
/tmp/autopkgtest.Gkv8A7/autopkgtest_tmp/test_xblockwise_reducer.cpp:3:10: fatal error: 'xtensor/xblockwise_reducer.hpp' file not found
#include "xtensor/xblockwise_reducer.hpp"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[3]: *** [CMakeFiles/test_xtensor_lib.dir/build.make:132: CMakeFiles/test_xtensor_lib.dir/test_xblockwise_reducer.cpp.o] Error 1
```

As far as I can see, this is because xblockwise_reducer.hpp is not listed in XTENSOR_HEADERS in CMakeLists.txt. So it never gets installed.

Is this a bug in CMakeLists.txt ?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -120,6 +120,8 @@ set(XTENSOR_HEADERS
     ${XTENSOR_INCLUDE_DIR}/xtensor/xassign.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xaxis_iterator.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xaxis_slice_iterator.hpp
+    ${XTENSOR_INCLUDE_DIR}/xtensor/xblockwise_reducer.hpp
+    ${XTENSOR_INCLUDE_DIR}/xtensor/xblockwise_reducer_functors.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xbroadcast.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xbuffer_adaptor.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xbuilder.hpp
@@ -151,6 +153,7 @@ set(XTENSOR_HEADERS
     ${XTENSOR_INCLUDE_DIR}/xtensor/xmasked_view.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xmath.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xmime.hpp
+    ${XTENSOR_INCLUDE_DIR}/xtensor/xmultiindex_iterator.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xnoalias.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xnorm.hpp
     ${XTENSOR_INCLUDE_DIR}/xtensor/xnpy.hpp

